protocol/v2/qbft/instance/instance.go

```diff
@@ -84,20 +88,35 @@ func (i *Instance) Start(value []byte, height specqbft.Height) {
 }
 
 func (i *Instance) Broadcast(msg *specqbft.SignedMessage) error {
-	byts, err := msg.Encode()
-	if err != nil {
-		return errors.Wrap(err, "could not encode message")
-	}
+	go func() {
```

Comment:
need to decide without breaking spec

Reply:
We returned back to the olds ways, because this broke spectests